scripts/ui/topological-ui-run.sh

```diff
@@ -0,0 +1,32 @@
+BOLD=$(tput bold)
+NORMAL=$(tput sgr0)
+
+export BASE_DIR=~/Projects
+
+# binding a dir into a container does not work when enforcing
+echo ${BOLD}Checking if Enforcing${NORMAL}
+getenforce | grep Enforcing && sudo setenforce 0
```

Comment:
Is it some condition for docker below? Is it necessary?

Reply:
see the comment above "binding a dir into a container does not work when enforcing"